charts/localstack/templates/role.yaml

```diff
@@ -19,10 +19,12 @@ rules:
 - apiGroups: [""]
   resources: ["services"]
   verbs: ["get", "list"]
-{{- if .Values.openshift }}
+{{- if .Values.scc }}
 - apiGroups: ["security.openshift.io"]
   resources: ["securitycontextconstraints"]
-  resourceNames: ["anyuid"]
+  resourceNames:
+    {{- range .Values.scc.resourceNames }}
+    - {{ . | quote }}
```

Comment:
I think it would be great if we could generalize this even a bit more by allowing arbitrary configs here, similar to the extraEnvVars here: 
https://github.com/localstack/helm-charts/blob/d8969409a40ea4114caf172350cef287f1a4b834/charts/localstack/templates/deployment.yaml#L188-L190

Something like `extraRoles` would allow us to enable the usage in OpenShift, while not committing to a specific derivate.
Waht do you think?

Reply:
Sounds good to me!  I will work on these changes.  Thanks for the feedback.

Reply:
Just so I understand, if the SCCs were to be created without OpenShift, do you mean something like Pod Security Policies, Pod Security Admission, or custom roles that basically mirror what the SCC would provide?

Reply:
I would really just "move the value template to one level higher", meaning that you would not set the resource names for a pre-defined / hard coded rule, but that you can add arbitrary complete rule entries.

So basically your value file would look something like this:
```
role:
  extraRules:
    - apiGroups: ["security.openshift.io"]
      resources: ["securitycontextconstraints"]
      resourceNames: ["anyuid"]
      verbs: ["use"]
```

We already do the same thing with the `extraEnvVars` linked above (where we even have template evaluation of the entries), so it should be fairly straight forward to implement.

And in the whole chart there would be no mention of OpenShift (because the chart would stay derivate-agnostic).
But it would still be great to have it explained how it works for OpenShift in the readme (thanks for the push on the docs)!

Reply:
Ok, I think i'm getting closer to what you are describing.  Let me know if you like this diff.